test/Concurrency/actor_derived_conformances.swift

```diff
@@ -0,0 +1,18 @@
+// RUN: %target-typecheck-verify-swift -disable-availability-checking
+// REQUIRES: concurrency
+
+actor A1: Comparable {}
+// expected-error@-1 {{type 'A1' does not conform to protocol 'Comparable'}}
+// expected-error@-2 {{type 'A1' does not conform to protocol 'Equatable'}}
+// expected-note@-3 {{automatic synthesis of 'Comparable' is not supported for actor declarations}}
+// expected-note@-4 {{automatic synthesis of 'Equatable' is not supported for actor declarations}}
+
```

Comment:
Would you mind adding a test for 

```
// RUN: %target-typecheck-verify-swift -disable-availability-checking
// REQUIRES: concurrency
// REQUIRES: distributed

import Distributed

distributed actor DA1: Comparable {}
```

it would fail the same way, but just to make sure we properly report it as "actor" as well; as discussed on chat, I think it's fine to report it as "actor" so no more implementation needed, just to make sure we report error nicely for it too.

Reply:
There's a `Distributed` directory right next to `Concurrency` where that test can live.

Reply:
Yup, can do!

Reply:
Done

Reply:
Thank you!